### ISSUE TITLE
error - enviroment unresolved identifier 

### ISSUE THREAD

**Issue by jobyid** [nodes-vapor/bugsnag#63]
Hi I am installing bugsnag into my vapor project, and I have gone throught the process, everything seems to have installed correctly but I have and error in the configure file. 
```
let bugsnagProvider = BugsnagProvider(config: BugsnagConfig(
        apiKey: "<apiKey>",
        releaseStage: environment.name, 
        shouldReport: environment.name != "local"
        debug: false
    ))
```
where enviroment appears I get an unresolved identifier error? 
Any ideas where I am going wrong? 
Many Thanks 
Joby

**Comment by siemensikkema** [comment]
`environment` refers to a variable of type `Environment` but the code sample in the doc is inconsistent. I just opened a PR to fix it: #64 In your case find out what the variable in question is named (could be `env` for example).

**Comment by jobyid** [comment]
Thanks, using env has worked.
